Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -575,7 +575,21 @@ void MainWindow::HideRenderWidget()
     disconnect(Host::GetInstance(), &Host::RequestTitle, this, &MainWindow::setWindowTitle);
     setWindowTitle(QString::fromStdString(Common::scm_rev_str));
   }
+
+  // The following code works around a driver bug that would lead to Dolphin crashing when changing
+  // graphics backends (e.g. OpenGL to Vulkan) to avoid this the render widget is (safely) recreated
```

Comment:
Nit: it feels like "to avoid this..." should be its own sentence.

Reply:
Done.